book_source/06_reference/01_pecan_standards.Rmd

```diff
@@ -5,8 +5,84 @@
 * New formats can be defined on the ['formats' page of BETYdb](http://betydb.org/formats)
 * After creating a new format, the contents should be defined by specifying the BETYdb variable name and the name used in the file/
 
-### Outputs
+### Input Standards
```

Comment:
The section below is specific to the input *meteorology* standard. This needs to be clearly indicated. Would also be good to include or reference the other input standards (exists for soil and veg pools, seems to be a bit of a WIP for the veg cohorts)